lib/abide_dev_utils/sce/generate/coverage_report.rb

```diff
@@ -364,9 +309,17 @@ def find_controls_in_mapping_data
               mapping.each do |level, profs|
                 next if level == 'benchmark'
 
-                profs.each do |_, ctrls|
-                  arr << ctrls.keys
-                  arr << ctrls.values
+                case @benchmark.framework
+                when 'cis'
+                  profs.each do |_, ctrls|
+                    arr << ctrls.keys
+                    arr << ctrls.values
+                  end
+                when 'stig'
+                  require 'pry'
+                  binding.pry
```

Comment:
Did you mean to leave this in?

Reply:
I did not, good catch. In fact, that whole method needs to go.